docs/release-notes/8.0.asciidoc

```diff
@@ -102,7 +102,7 @@ A new Lucene 9 validation change may cause event correlation (EQL) rule errors w
 [discrete]
 [[breaking-changes-8.0.0]]
 ==== Breaking Changes
-// tag::breaking-changes[]
+
 :pull: https://github.com/elastic/kibana/pull/
```

Comment:
```suggestion
```

Reply:
I may be missing something, but I think this attribute gets used below. Let me know if I'm wrong! 😅

Reply:
No worries! Our `issue` and `pull` attributes are actually defined at a docset level in [release-notes.asciidoc](https://github.com/elastic/security-docs/blob/main/docs/release-notes.asciidoc?plain=1#L39), so it should be fine to remove the `pull` definitions inside the breaking changes sections. @joepeeples feel free to correct me if I'm wrong about this.

Reply:
Ah! Thanks — I'll remove 'em.